jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/identity/JBossUserGroupCallbackImpl.java

```diff
@@ -99,7 +99,7 @@ public boolean existsGroup(String groupId) {
 		if( groups == null ) { 
 		    groups = new ArrayList<String>(0);
 		}
-		return groups;
+		return new java.util.ArrayList<String>(groups);
```

Comment:
Maybe remove the package name as it is already in the import statements.

Reply:
Done. Thanks!